qemu/tests/check_link_speed_duplex.py

```diff
@@ -0,0 +1,148 @@
+import logging
+import re
+
+from virttest import error_context
+from virttest import utils_net
+from virttest import utils_test
+from virttest import env_process
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    """
+    def get_speed_duplex_traceview():
+        """
```

Comment:
There is no need for this empty doc string.

Reply:
I added doc string for this scripts now, pls help to review it again.